rules/0250-apache_rules.xml

```diff
@@ -221,9 +218,6 @@
     <description>Apache: Client sent malformed Host header. Possible Code Red attack.</description>
     <info type="link">http://www.cert.org/advisories/CA-2001-19.html</info>
     <info type="text">CERT: Advisory CA-2001-19 "Code Red" Worm Exploiting Buffer Overflow In IIS Indexing Service DLL</info>
```

Comment:
Same as above: For this one the tags T1190 "Exploit Public-Facing Application" and T107.001 "Application Layer Protocol: Web Protocols" may be correct